src/services/wallets/tweb3.ts

```diff
@@ -42,11 +42,12 @@ const popupStyling = {
 // interface TorusParams in trous-embeded types
 const initSettings = (buildEnv: BuildEnvironment) => {
   return {
-    buildEnv: buildEnv || "production",
+    buildEnv:
+      (process.env.BUILD_ENV as BuildEnvironment) || buildEnv || "production",
     network: {
-      host: "http://localhost:7545",
-      chainId: 1337,
-      networkName: "Localchain",
+      host: process.env.CHAIN_HOST || "http://localhost:7545",
+      chainId: Number(process.env.CHAIN_ID) || 1337,
+      networkName: process.env.CHAIN_NAME || "Localchain",
```

Comment:
maybe keep these in one place an import a config object?

Reply:
I thought about making a JSON config file and just importing it as an object. I ended up going with the .env file since it seemed to be more widely used for these types of global constants. I also didn't want to create them to be specific to Torus. It's likely that later on things like CHAIN_ID and CHAIN_HOST may need to be used elsewhere.